frontend/package.json

```diff
@@ -18,13 +18,23 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "lint": "eslint .",
```

Comment:
will it ignore `node_modules/`?
may be add lint `eslint src/`?

Reply:
done